src/treetensornetworks/solvers/tdvp.jl

```diff
@@ -122,17 +68,18 @@ function tdvp(
   (sweep_observer!)=observer(),
   root_vertex=default_root_vertex(init),
   reverse_step=true,
+  updater_kwargs=NamedTuple(;),
   kwargs...,
 )
   nsweeps = _compute_nsweeps(nsteps, t, time_step, order)
-  sweep_regions = tdvp_sweep(order, nsite, time_step, init; root_vertex, reverse_step)
+  region_updates = tdvp_sweep(order, nsite, time_step, init; root_vertex, reverse_step)
```

Comment:
```suggestion
  sweep_plan = tdvp_sweep_plan(order, nsite, time_step, init; root_vertex, reverse_step)
```
Maybe, up for discussion.